packages/youtube_player_iframe/assets/player.html

```diff
@@ -147,6 +149,15 @@
           }
         }
       }
+
+      function handleMute() {
+        setInterval(() => {
+          if (muted != player.isMuted()) {
+            sendMessage('MuteButtonPressed', player.isMuted());
+            muted = player.isMuted();
+          }
+        }, 500);
+      }
```

Comment:
Since the controller already provides access to `isMuted`, you can do the same thing already on Flutter side.

Reply:
Hey, the isMuted is also buggy, since you need to check if isMuted is string true or false, you are checking with '1'. Also rather than timer in flutter, we have added same logic in javascript.

Reply:
Then i guess it'll be better to fix `isMuted`. Any specific reason to have the timer in js side ?

Reply:
If is isMuted is fixed, we can do the same in flutter, but it would be better to have the functionality in js, such that any change, calls from js to dart once, rather than dart calling to js every time. Also it would be better functionality.